### PR TITLE
[ECO-283] Change decimals and name of faucet test currency

### PR DESCRIPTION
**Pull request by elliottdehn**
tETH has 18 decimals and thus you can only have 18 whole units of tETH before overflowing the u64. Let's use a different currency (tAPT/TestAPT) with 8 decimals so that this lines up better. Everyone who's run this code has run into this issue.

It might be kind of confusing to have tAPT and APT adjacent to one another, but I think it's ok.

### CODE REVIEW

**Review by bogberry — CHANGES_REQUESTED** [review]
Why not just use testnet APT?

**Comment by elliottdehn** [comment]
> Why not just use testnet APT?

We're limited in how many APT we can request to mint, so it's much easier to be able to request lots fake APT than 1 APT at a time from a "real" faucet.

**Review by alnoki — CHANGES_REQUESTED** [review]
As just discussed during a team call, let's switch to `eAPT` ("Econia APT") and `eUSDC` ("Econia USDC") to avoid confusion that might arise from the `t` prefix (e.g. Tortuga liquid staking derivative)

File: src/python/sdk/examples/trade.py [hunk 1]
```diff
@@ -127,8 +127,8 @@ def get_aptos_faucet_url() -> str:
     get_econia_address()
 )  # See https://econia.dev/ for up-to-date per-chain addresses
 FAUCET_ADDR = get_faucet_address()  # See (and deploy): /econia/src/move/faucet
-COIN_TYPE_ETH = f"{FAUCET_ADDR}::test_eth::TestETH"
-COIN_TYPE_USDC = f"{FAUCET_ADDR}::test_usdc::TestUSDC"
+COIN_TYPE_EAPT = f"{FAUCET_ADDR}::example_apt::ExampleAPT"
+COIN_TYPE_USDC = f"{FAUCET_ADDR}::example_usdc::ExampleUSDC"
```
Comment (alnoki) [inline src/python/sdk/examples/trade.py:131]: For consistency how about `COIN_TYPE_EUSDC`?

**Review by alnoki — CHANGES_REQUESTED** [review]

**Review by alnoki — CHANGES_REQUESTED** [review]

**Review by alnoki — CHANGES_REQUESTED** [review]

File: src/move/faucet/sources/example_apt.move [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
     use aptos_std::string;
     use econia_faucet::faucet;
 
-    struct TestETH {}
+    struct ExampleAPT {}
 
-    const NAME: vector<u8> = b"Test Ether";
-    const SYMBOL: vector<u8> = b"tETH";
-    const DECIMALS: u8 = 18;
+    const NAME: vector<u8> = b"Example Aptos coin";
```
Comment (alnoki) [inline src/move/faucet/sources/example_apt.move:7]: "Example Aptos Coin" (capital C)

**Review by alnoki — CHANGES_REQUESTED** [review]

**Review by alnoki — APPROVED** [review]

**Review by bogberry — APPROVED** [review]
Yup all good

**Pull request merged**